uiowa/uiowa
# remove permissions related to user cancellation

Due to issues with cancelling accounts (#4761 and [moderation revisions](https://www.drupal.org/project/drupal/issues/3089747)), we are removing the ability for webmasters to cancel accounts. Instead we recommend blocking the account.

In the future, we may be able to make this available again.

Content authored by a blocked user must be reassigned for successful save. See https://github.com/uiowa/uiowa/issues/6960

# To Test

```
ddev blt ds --site=sandbox.uiowa.edu
```

As webmaster, you no longer see the ability to cancel a user account. The option is still in the bulk operations, but you get an access denied message on execute.

<img width="722" alt="Screenshot 2024-01-10 at 10 00 41 AM" src="https://github.com/uiowa/uiowa/assets/4663676/dc384597-8622-4391-824d-576e15cd2038">
